lib/base_service.ts

```diff
@@ -14,12 +14,14 @@
  * limitations under the License.
  */
 
- 'use strict';
+'use strict';
```

Comment:
do we need the `use strict` ?

Reply:
This doesn't need to be here. I can configure the `tsc` compiler to emit `use strict` in the .js files.